src/zoid/buttons/component.jsx

```diff
@@ -42,6 +42,17 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
         logger: getLogger(),
 
         prerenderTemplate: ({ state, props, doc }) => {
+            const { buttonSessionID } = props;
```

Comment:
Looks good. I think you could also log this logic in the onInit() decorator on line 260.

Reply:
@gregjopa I had originally it there but wanted it as early as possible.  The reason is my next update will suppress the Venmo button (on prerender) if `localStorage` is not accessible.